adopters/1-flux-v2.yaml

```diff
@@ -18,3 +18,6 @@ adopters:
     - name: Nexxiot
       url: https://www.nexxiot.com
       logo: logos/nexxiot.png
+    - name: Yad2
+      url: https://www.yad2.co.il
+      logo: logos/yad2.png
```

Comment:
Please also add the logo to the entry in `3-flux-v1.yaml`: https://github.com/fluxcd/website/blob/main/adopters/3-flux-v1.yaml#L165-L166